flume/conf/agent.conf.template

```diff
@@ -15,7 +15,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 
 #=============================================
-# To be put in APACHE_FLUME_HOME/conf/cygnus.conf
```

Comment:
NTC

Yes, this PR solves some issues introduced in PR [#307](https://github.com/telefonicaid/fiware-connectors/pull/307)